modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/authorized_as_representative_spec.rb

```diff
@@ -69,14 +57,8 @@
 
       context 'and RepresentativeUserAccount raises Forbidden (e.g., OGC conflict/none)' do
```

Comment:
```suggestion
      context 'and PowerOfAttorneyHolderMemberships raises Forbidden (e.g., OGC conflict/none)' do
```

Reply:
It gets nuked in a later PR! As for this change suggestion, I'll save it in my list of changes to apply at the tail of the PR stack.